logging/measure/model/apitrace/apitrace.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+// Package apitrace provides data model for binary API trace informations.
+package apitrace;
+
+message Trace {
+    message Entry {
+        uint64 index = 1;                       // Trace index. Every message is indexed starting with 1 in order they
+                                                // called in in the VPP
+        string msg_name = 2;                    // Message name (or any other descriptive string)
+        uint64 duration = 3;                    // Duration of message call in nanoseconds
+    }
```

Comment:
Entries should definitely contain start time as well so we can later filter them and see how close they are to each other.